src/layouts/Navbar.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Link from "@material-ui/core/Link";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+  navbar: {
+    justifyContent: "space-evenly",
+    alignItems: "center",
+    backgroundColor: "#343B41",
+    color: "#F7F7F7",
+    height: "40%",
+    textTransform: "uppercase",
+    fontWeight: "bold",
+    "&:hover": {
+      textDecoration: "none",
+    },
+  },
+  navlink: {
+    "&:hover": {
+      textDecoration: "none",
+    },
+  },
+}));
+
+function Navbar() {
+  const classes = useStyles();
+  const preventDefault = (event) => event.preventDefault();
+  const links = [
```

Comment:
could be extracted outside

Reply:
Extracted links outside, removed preventDefault and because of already used Link component.